learn/configuration/instance_options.md

```diff
@@ -4,10 +4,101 @@ sidebarDepth: 2
 
 # Configure Meilisearch at launch
 
-You can configure Meilisearch at launch with **environment variables** and **command-line options**.
+You can configure Meilisearch at launch with **command-line options**, **environment variables**, or a **configuration file**.
 
 These startup options affect your entire Meilisearch instance, not just a single index. For settings that affect search within a single index, see [index settings](/learn/configuration/settings.md).
 
+## Configuration file
+
+Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
+
+**Environment variables can overwrite the configuration file, and command-line options can overwrite environment variables.**
+
+You can download the configuration file using the following command:
+
+<CodeSamples id="instance_options_config_file_download_1" />
+
+Once downloaded, Meilisearch will use it as the default configuration file. You will see it when you launch Meilisearch:
+
+```
+888b     d888          d8b 888 d8b                                            888
+8888b   d8888          Y8P 888 Y8P                                            888
+88888b.d88888              888                                                888
+888Y88888P888  .d88b.  888 888 888 .d8888b   .d88b.   8888b.  888d888 .d8888b 88888b.
+888 Y888P 888 d8P  Y8b 888 888 888 88K      d8P  Y8b     "88b 888P"  d88P"    888 "88b
+888  Y8P  888 88888888 888 888 888 "Y8888b. 88888888 .d888888 888    888      888  888
+888   "   888 Y8b.     888 888 888      X88 Y8b.     888  888 888    Y88b.    888  888
+888       888  "Y8888  888 888 888  88888P'  "Y8888  "Y888888 888     "Y8888P 888  888
+
+Config file path:       "./config.toml"
+```
+
+If no configuration file is provided, the `Config file path` is set to `none`.
```

Comment:
I would put it at the end of the section, because if no `./config.toml` file are found OR if you haven't provided any file with `--config-file-path`, `none` is displayed.
This sentence only helps users for debugging but is not necessary to understand the feature.